src/__swaps__/screens/Swap/components/ReviewPanel.tsx

```diff
@@ -159,6 +159,21 @@ function FlashbotsToggle() {
   );
 }
 
+function DegenModeToggle() {
+  const { SwapSettings } = useSwapContext();
+
+  const degenMode = useDerivedValue(() => SwapSettings.degenMode.value);
```

Comment:
I *think* we can nix this derived value and just consume the `SwapSettings.degenMode` in the AnimatedSwitch below.